### ISSUE TITLE
Verbose mode

### ISSUE THREAD

**Issue by fliptheweb** [elwin013/parcel-plugin-static-files-copy#58]
Hi!
Problem:
Plugin produces logs to console on each build in development, like
```js
Static file '...' already exists in '...'. Overwriting.
```
But in most my cases that's totally unnecessary.

Do you mind if I add the `verbose` option (default is `true`) to settings for turn off such messages?

**Comment by imaegoo** [comment]
+1

I got 100+ lines console log when I start parcel dev server and change codes.

I expected that static file should not overwrite when their modified time are same.

``` diff
- if (destStat.mtime <= srcStat.mtime) { // File was modified - let's copy it and inform about overwriting.
+ if (destStat.mtime < srcStat.mtime) { // File was modified - let's copy it and inform about overwriting.
```

![WeChat Screenshot_20200727132718](https://user-images.githubusercontent.com/20182252/88506707-1c874200-d00d-11ea-95c7-13b2805106ef.png)

**Comment by elwin013** [comment]
Hey @fliptheweb 

> Do you mind if I add the verbose option (default is true) to settings for turn off such messages?

Plugin is writing to console on "level 3" (info) and "level 2" (warning, only once, when there is no static dir). Does setting ParcelJS log level to 2 is ok for you (see https://parceljs.org/cli.html#change-log-level)?

**Comment by fliptheweb** [comment]
@elwin013 thank you for showing me such options!
But I guess that @imaegoo fix will be pretty enough for my needs.
Have a good Friday for both of you :)

**Comment by elwin013** [comment]
@fliptheweb Thanks! :-) 

I've just published v2.4.4 which contains @imaegoo fix. Please let me now if that works for you. :-)

**Comment by elwin013** [comment]
Closing :-)
